### ISSUE TITLE
Discoid adrenal gland

### ISSUE THREAD

**Issue by peupeubangbang** [obophenotype/human-phenotype-ontology#10938]
**Preferred term label:**
Discoid adrenal gland

**Synonyms**
Straight adrenal gland, elongated adrenal gland, pancake adrenal gland, lying down adrenal gland, lying down adrenal sign

**Definition (free text, please give PubMed ID)**
A congenital finding where the adrenal gland does not form it's typical triangular, Y or V shape and is instead discoid, straight or elongated in shape. It is often found incidentally on renal ultrasound and is associated with renal agenesis and other renal congenital anomalies.
PMID: 33552228

**Parent term (use hpo.jax.org/app)**
Abnormal adrenal morphology HP:0011732

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Renal agenesis ORPHA:411709

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@peupeubangbang 

Adding new term: 
- ID: HP:6001167
- label: Discoid adrenal gland
- parents: [Abnormal adrenal morphology (HP:0011732)](https://hpo.jax.org/app/browse/term/HP:0011732)
- pmid: PMID:33552228
- def: A congenital finding where the adrenal gland does not form it's typical triangular, Y or V shape and is instead discoid, straight or elongated in shape. It is often found incidentally on renal ultrasound and is associated with renal agenesis and other renal congenital anomalies.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10938
- synonyms: Elongated adrenal gland(NONE)|Lying down adrenal sign(NONE)|Straight adrenal gland(NONE)|Lying down adrenal gland(NONE)|Pancake adrenal gland(NONE)
